README.md

```diff
@@ -3,33 +3,89 @@
 [![All Contributors](https://img.shields.io/badge/all_contributors-5-orange.svg?style=flat-square)](#contributors-)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
 
-This action generates whatever you want using your AsyncAPI document. It uses [AsyncAPI Generator](https://github.com/asyncapi/generator/).
+This action exposes the [AsyncAPI CLI](https://github.com/asyncapi/cli). It allows you to generate documentation, validate AsyncAPI documents, convert between different AsyncAPI versions and much more.
 
 ## Inputs
 
+### `cli_version`
+
+Version of the AsyncAPI CLI you wish to use. You can find all available versions [here](https://github.com/asyncapi/cli/releases). Recommended leave it out of the inputs and use the default value.
+
+**Default** points to the`latest` version.
+
+> [!TIP]
+> We recommend to default to `latest` version. This way there is no overhead with the script updating the CLI version. As it takes a lot of time to update the CLI version, we recommend to update it only when you need to use another one for compatibility reasons.
+
+### `command`
+
+Command that you wish to run. You can find all available commands Available commands are:
+- `generate` - generates documentation from AsyncAPI document
+- `validate` - validates AsyncAPI document
+- `optimize` - optimizes AsyncAPI document
+- `custom` - allows you to run any command that is available in the AsyncAPI CLI. You can find all available commands [here](https://www.asyncapi.com/docs/tools/cli/usage).
```

Comment:
I'm still not sure what is the goal of custom command solution 🤔 why cannot we have all commands here?

Reply:
The problem is we would have to make too many switch statement as it is not one format for all commands so I have some basic commands so that the existing ones won't break.